src/features/about/ExperienceTimeline.tsx

```
@@ -0,0 +1,1139 @@
+//       </div>
+//     </section>
+//   );
+// }
+
+
+
+import { useState, useRef, useEffect } from 'react';
+import { motion, AnimatePresence, useScroll } from 'framer-motion';
+import { ArrowRight } from 'lucide-react';
+
+export const experiences = [
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)